### ISSUE TITLE
Apple Carplay issues

### ISSUE THREAD

**Issue by DrZere** [txthinking/brook#1361]
Spotify and call audio is not playing at all when connected to carplay and brook is running
Sometimes carplay wont connect when brook is connected and the issue goes away after disconnecting brook.

**Comment by txthinking** [comment]
Yes, I Googled it and found that it is indeed a common phenomenon. I think I should try to solve it in the next version.

**Comment by DrZere** [comment]
I think the problem is the vpn blocking local connections. Sometimes when i connect to my car i see an ethernet option in my settings.

**Comment by txthinking** [comment]
try v20250202
